### DISCUSSION TITLE
Scrubabdub is awesome but...

### DISCUSSION THREAD

**Discussion by ian-lewis-d** [LeapBeyond/scrubadub#136] [General]
As a native English speaker  the use of the word 'filth' in Scrubadub feels like a bad choice. I totally get it, the task is to _clean_ data of PII/GDPR related issues. 'Filth' has connotations of organic muck and dirt and sounds almost childish.

**Comment by deanmalmgren** [comment]
Fair point @ian-lewis-d. Admittedly, as the original author of this package, I did get a chuckle out of naming things `Filth`, but if you would like to propose a better name — especially with a PR, we'd welcome it! 

On the one hand, I agree with you that probably there is a more professional sounding name. But on the other, the package is called `scrubadub` which isn't exactly the most professional of names to begin with either 😂
